tests/store/test_status_analyses_filters.py

```diff
@@ -180,9 +178,7 @@ def test_filter_analyses_without_delivery_report(base_store: Store, helpers: Sto
     assert analysis_without_delivery_report in analyses
 
 
-def test_filter_report_analyses_by_pipeline(
-    base_store: Store, helpers: StoreHelpers, case_obj: models.Family
-):
+def test_filter_report_analyses_by_pipeline(base_store: Store, helpers: StoreHelpers, case_obj):
```

Comment:
No type hint?

Reply:
Probably a mistake when i resolved the giant merge conflict 😅